src/platform_impl/ios/event_loop.rs

```diff
@@ -56,9 +57,10 @@ impl<T: 'static> EventLoopWindowTarget<T> {
         unsafe { monitor::uiscreens() }
     }
 
-    pub fn primary_monitor(&self) -> MonitorHandle {
+    pub fn primary_monitor(&self) -> Option<RootMonitorHandle> {
         // guaranteed to be on main thread
-        unsafe { monitor::main_uiscreen() }
+        let inner = unsafe { monitor::main_uiscreen() };
+        Some(RootMonitorHandle { inner })
```

Comment:
```suggestion
        let monitor = unsafe { monitor::main_uiscreen() };
        Some(RootMonitorHandle { inner: monitor })
```

I think this naming would be slightly better and prevent confusions with potential future refactorings? But it doesn't matter that much, especially since using `inner` makes the return shorter.